data.sql

```diff
@@ -11,3 +11,24 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
 
```

Comment:
## 1
![image](https://user-images.githubusercontent.com/93039359/213637616-125183ca-692a-42fc-a45a-12b3eb4112af.png)
## 2
![image](https://user-images.githubusercontent.com/93039359/213637703-2e01f1c3-3afb-4fe3-a8eb-418c82872b56.png)
## 3
![image](https://user-images.githubusercontent.com/93039359/213637760-9c95f5e6-ee67-4681-adc3-746bdea2132b.png)
## 4
![image](https://user-images.githubusercontent.com/93039359/213637854-fd618be3-6014-458c-abaf-3c18990a7f65.png)

- [ ] Your readme file is not really good-looking. Kindly remove the sections which I have shown in the images above arrow_up: to make it descriptive and professional :+1: